actionview/lib/action_view/helpers/tag_helper.rb

```diff
@@ -50,68 +50,20 @@ def initialize(view_context)
         def tag_string(name, content = nil, escape_attributes: true, **options, &block)
           content = @view_context.capture(self, &block) if block_given?
           if VOID_ELEMENTS.include?(name) && content.nil?
-            "<#{name.to_s.dasherize}#{tag_options(options, escape_attributes)}>".html_safe
+            tag_attributes = @view_context.tag_attributes(options, escape_attributes)
+            "<#{name.to_s.dasherize}#{tag_attributes ? " #{tag_attributes}" : nil}>".html_safe
```

Comment:
I think we can extract this conditional of `tag_attributes ? :` into a `tag_attributes` method on this class so we can keep this code almost the same.

Reply:
Thanks for the feedback! I've added the method and rebased.